interfaces/Portal/src/app/program/program-people-affected/program-people-affected.component.ts

```diff
@@ -995,8 +998,6 @@ export class ProgramPeopleAffectedComponent implements OnDestroy {
   ) {
     const statusRelatedBulkActions = [
       BulkActionId.include,
```

Comment:
I think we need to add somewhere to the bulk actions that you can `Decline` registrations. Probably in the payments phase. I created this task for it AB#27875

Reply:
I mean this menu: 
![image](https://github.com/global-121/121-platform/assets/34537157/0fd6d8e6-1928-4520-a466-dac4f183cda3)

Reply:
Updated the PR!
it was one flag `showIfNoValidation` which was `false` and due to this it was not showing the menu option `Mark as Declined`. I made it `true`. Could you please confirm that its the right change. Thanks!